apps/frontend/src/app/shared/components/layout/navbar-pages/navbar-pages.component.html

```diff
@@ -252,47 +252,52 @@
               aria-labelledby="menu"
             >
               <div class="py-2">
-                <div
-                  class="hover:bg-primary w-full text-base-200 hover:text-base-100 p-2 hover:rounded-lg"
+                <a
```

Comment:
I found a strange behavior here, the logout hover goes all the way to the end of the div while the other don't. We should standardize this. 

![image](https://github.com/Venture-Miner/wagademy/assets/111710515/9c043983-4649-44eb-a1e2-6e3654603f4f)


![image](https://github.com/Venture-Miner/wagademy/assets/111710515/81d58e03-d99b-454c-aa5a-292caff0673e)